dex/testing/eth/create-node.sh

```diff
@@ -135,8 +132,10 @@ if [ "${SYNC_MODE}" = "snap" ]; then
   tmux send-keys -t "$TMUX_WIN_ID" "${NODES_ROOT}/harness-ctl/${NAME} --nodiscover " \
 	  "--config ${NODE_DIR}/eth.conf --unlock ${CHAIN_ADDRESS} " \
 	  "--password ${GROUP_DIR}/password --light.serve 25 --datadir.ancient " \
-	  "${NODE_DIR}/geth-ancient --verbosity 5 --vmdebug 2>&1 | tee " \
-	  "${NODE_DIR}/${NAME}.log" C-m
+	  "${NODE_DIR}/geth-ancient --verbosity 5 --vmdebug --http --http.port " \
+	  "${HTTP_PORT} --ws --ws.port ${WS_PORT} --ws.api " \
+	  "eth --allow-insecure-unlock " \
```

Comment:
Removed all the modules except eth. This should be all we need. Also "db" was apparently not a module? This was for the light client harness node.
```
ERROR[02-03|11:23:46.895] Unavailable modules in HTTP API list     unavailable=[db] available="[admin debug web3 eth txpool personal clique net les vflux engine]"
```

Reply:
Noticed this too: https://github.com/decred/dcrdex/pull/2013#discussion_r1088346355
No clue why it was ever there.  Good to have it here too.

However, didn't we still want txpool? Is the idea that we want it to use the dumbed down fallback path?  Or that beta is just used for server?

Reply:
> However, didn't we still want txpool? Is the idea that we want it to use the dumbed down fallback path? Or that beta is just used for server?

I guess that server operators will use providers, either by choice or because the eth node can not be kept up %100. I also guess that the txpool type operations will not be available for providers. Maybe we can turn txpool on for one and off for the other? So we can use both for testing.

Reply:
This is the simnet harness, so I mean if you're testing dexc and want to use the beta node?  Would that just be pointless and now expected not to work?  I'm fine with that, if that's intended.

Reply:
> This is the simnet harness, so I mean if you're testing dexc (client, not server) and want to use the beta node? Would that just be pointless and now expected not to work?

I didn't think the client needed anything but "eth" and only server needed "txpool" (doesn't need now)

I think client is fine with with either "eth" or "eth,txpool"?

Reply:
Now alpha is `eth` and beta is `eth,txpool`